### DISCUSSION TITLE
ione-api.xxxxxx shows No Credentials given

### DISCUSSION THREAD

**Discussion by Muhammad-Aqueel** [ione-cloud/ione#199] [General]
Hi,
I need your help to resolve this.

when I access **_ione-api.xxxxxxxx_**
page shows this text only
No Credentials given

Thanks

**Comment by slntopp** [comment]
Hi, this is an API, you should use it for calls. In order to provide Credentials add `Authorization` header.
If you're seeking for AdminUI, it's statically served and by default should be accessible at _**ione-admin.xxxxxxxx**_

**Reply by Muhammad-Aqueel** [reply]
I am able to login to ione-admin.xxxxxxxx using sunstone user and password but when I access ione-api.xxxxxxxx No Credentials given show only on page.
How to add Authorization header to to ion-api ?
Please assist me.

**Reply by slntopp** [reply]
It's an quasi-REST API, you can access it via Postman or your programming language.
Example requests can be found here as [Postman](https://documenter.getpostman.com/view/5695180/UVsEU8R6)

**Comment by Muhammad-Aqueel** [comment]
How to get these functionalities demonstrated on iOne webpage?
![image](https://user-images.githubusercontent.com/79786283/157018623-b7cd3a07-fe40-48c5-ac4b-bceff43db41b.png)

**Reply by slntopp** [reply]
You should leave your inquiry at the website. You can install it by yourself with our instructions, though you'll need WHMCS and our module for it, or get our solution and we'll install everything for you.

**Reply by Muhammad-Aqueel** [reply]
when I select Addon > opennebula control this error displayed.
![image](https://user-images.githubusercontent.com/79786283/157134421-a56b653f-5476-44c1-aa81-e4ee284b9e6d.png)

**Reply by slntopp** [reply]
Hi, we just pushed a patch, could please try updating module to the latest code

**Reply by Muhammad-Aqueel** [reply]
ok let me check

**Reply by Muhammad-Aqueel** [reply]
Still facing the same error.

**Reply by Muhammad-Aqueel** [reply]
WHMCS 8.2.0 I am using
